bin/shpec

```diff
@@ -107,6 +107,18 @@ assert() {
     print_result "$2" \
       "Expected $2 to be true"
   ;;
+  ( xgrep )
+    print_result "echo \"$2\" | grep -q \"$3\"" "Expected $2 to match $3"
```

Comment:
For consistency with our other output (and, more importantly, for easier parsing by the user), we should wrap the arguments in `[]` here.